app/views/posts/index.html.erb

```diff
@@ -14,7 +14,7 @@
 
   <div class="posts__post">
     <% @posts.each_with_index do |post, index| %>
-      <a href="posts/<%= post.id %>">
+      <%= link_to user_post_path(@user, post.id) do %>
         <div class="posts__post--li">
           <h3>Post #<%= index + 1 %></h3>
```

Comment:
- [ ] Kindly fix this part of the code, it is commented out(#) and has introduced a bug in our code base, hence the `<h3>` is not closed